server/controllers/stock/reports/stock_exit_depot.receipt.handlebars

```diff
@@ -20,8 +20,10 @@
             {{translate 'STOCK.RECEIPT.EXIT_DEPOT'}}
           </span> <br>
           <strong>{{exit.details.depot_name}}</strong> <br>
-          <strong>{{exit.details.document_reference}}</strong> <br>
-          <small>{{date exit.details.date}}</small> <br>
+          <div class="row">
+            <small>{{> barcode value= exit.details.barcode}}</small>
+            <br/><br/>
```

Comment:
Could you remove this `<br/><br/>`, i think it is not necessary to have these spaces